go.work

```diff
@@ -5,6 +5,7 @@ use (
 	./api
 	./client/v2
 	./core
+	./cosmovisor
```

Comment:
I think we decided to remove this? @julienrbrt you did something related to this no?

Reply:
It will likely need to be re-removed, then its removal will cause some kind of issue I don't fully understand.

Reply:
Your issue was in go 1.19 you said right? So it won't happen in this issue anyway :)

Reply:
https://github.com/cosmos/cosmos-sdk/pull/12851#discussion_r941885873